docscss/docs.css

```diff
@@ -67,31 +74,60 @@
 }
 
 .topbar {
-    padding: 1.7em 1.4em;
-    border-bottom: 1px solid var(--border-color);
+    padding: 1.45em 1.4em;
+    /* border-bottom: 1px solid var(--border-color); */
     display: flex;
-    justify-content: flex-end;
+    justify-content: space-between;
     align-items: center;
-    box-shadow: 0 2px 2px -2px rgba(0, 0, 0, .2);
+    box-shadow: var(--box-shadow);
+    /* position: sticky;
+    right: 0;
+    left: 0;
+    top: 0; */
```

Comment:
it'll be better if you remove the commented styles to have a clean looking stylesheet

Reply:
I commented some lines for more work.

Reply:
oh in that case it's fine then! btw great work👍